aspnetcore/blazor/globalization-localization.md

```diff
@@ -1618,7 +1618,7 @@ To create localization shared resources, adopt the following approach.
   * `Localization/SharedResource.es.resx`
 
   > [!NOTE]
-  > `Localization` is resource path that can be set via <xref:Microsoft.Extensions.Localization.LocalizationOptions>.
+  > Use <xref:Microsoft.Extensions.Localization.LocalizationOptions.ResourcesPath%2A?displayProperty=nameWithType> to set a different path when shared resource files aren't in the same folder as the dummy class. **Supply a path relative to the location of the dummy class, not the root of the app.**
```

Comment:
It's not necessary a shared resource, right?

Reply:
Yes, but this section only applies to shared resources, and the remark is only addressing the guidance provided here.

Reply:
I think there's no much difference between shared or non-shared resource.
What matters is where resx file is located relative to the class and whether class exists at all.

Reply:
It seems to me like a design problem, and I encourage someone ... not necessarily either of you ... to open a product unit issue on it. For now, I think I can get the article's remark correct. I'll try an update shortly.